VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -140,6 +186,7 @@ def get_total_used_size(self):
             index = 1
             self.resource_disk= ResourceDiskUtil(patching = self.patching, logger = self.logger)
```

Comment:
Let's add a comment that ResourceDisks are excluded in billing as they are similar to temporary storage